crypto/cipher_extra/cipher_extra.c

```diff
@@ -106,6 +106,7 @@ static const struct {
   const char* name;
 } kCipherAliases[] = {
```

Comment:
What uses these aliases? Does adding DES here enable it to be used silently by default anywhere?

Reply:
1. Adding this mapping does not "leak" anything new. We already expose des-cbc through the kCiphers array. This change adds another alias pointing "DES" to this mapping (cbc is the default mode for DES in OpenSSL). 
2. These aliases are checked in addition to the kCiphers array. This is due to differences in the string names applications try to fetch algorithms with. In general these string:EVP_CIPHER mappings are used by the EVP_get_cipherbyname function. This function is part of our public API, although deprecated, which means users can directly retrieve the EVP_des_cbc cipher object and then conduct operations with it. Internally I don't see any usages in our codebase other than testing.